pubsub/pullstream.go

```diff
@@ -47,6 +48,7 @@ func newPullStream(ctx context.Context, streamingPull streamingPullFunc, subName
 			spc, err := streamingPull(ctx, gax.WithGRPCOptions(grpc.MaxCallRecvMsgSize(maxSendRecvBytes)))
 			if err == nil {
 				recordStat(ctx, StreamRequestCount, 1)
+				log.Print("starting open")
 				err = spc.Send(&pb.StreamingPullRequest{
 					Subscription: subName,
 					// We modack messages when we receive them, so this value doesn't matter too much.
```

Comment:
Immediately after the spc.Send call, add

`log.Printf("open returned %v", err)`